content/assignments/minishell/src/utils.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#ifndef _UTILS_H
+#define _UTILS_H
+
+#include "../util/parser/parser.h"
+
+
+/* Useful macro for handling error codes. */
+#define DIE(assertion, call_description)  \
+	do {                                    \
+		if (assertion) {                      \
+			fprintf(stderr, "(%s, %s, %d): ",   \
+				__FILE__, __func__, __LINE__);    \
+			perror(call_description);           \
+			exit(EXIT_FAILURE);                 \
+		}                                     \
```

Comment:
```suggestion
#define DIE(assertion, call_description)			\
	do {							\
		if (assertion) {				\
			fprintf(stderr, "(%s, %s, %d): ",	\
				__FILE__, __func__, __LINE__);	\
			perror(call_description);		\
			exit(EXIT_FAILURE);			\
		}						\
```

Reply:
I don't understand the issue here.
They are already aligned.

Reply:
No, they're not. Most probably you haven't set your tab width to 8, most likely to 4. There were 2 issues:
- you used spaces instead of tabs
- they were not aligned (considering 8-byte wide tabs)

Reply:
Actually, I have tab width to 2 because I don't want to waste screen space for anything.
Why should I use tabs though? Spaces are used for alignment.

Reply:
>  I have tab width to 2 because I don't want to waste screen space for anything.

@gabrielmocanu also known as "mister efficiency". Use tabs because they're Linux Kernel standard and take up fewer bytes.